components/econet/econet.cpp

```diff
@@ -155,31 +199,23 @@ void Econet::make_request() {
     return;
   }
 
-  if (model_type_ != MODEL_TYPE_HVAC || !hvac_wifi_module_connected_) {
-    std::vector<std::string> str_ids(datapoint_ids_.begin(), datapoint_ids_.end());
-    // TODO: Better handle RAW that likely need to be requested separately.
-    str_ids.erase(std::remove(str_ids.begin(), str_ids.end(), "AIRHSTAT"), str_ids.end());
-    request_strings(dst_adr, src_adr, str_ids);
-  }
+  std::vector<std::string> str_ids(datapoint_ids_.begin(), datapoint_ids_.end());
+  // TODO: Better handle RAW that likely need to be requested separately.
+  str_ids.erase(std::remove(str_ids.begin(), str_ids.end(), "AIRHSTAT"), str_ids.end());
```

Comment:
This line removes AIRHSTAT from the datapoints to request because I believe RAW datapoints need to be requested on their own. At least that's how the the ACK command is implemented.
And AIRHSTAT is added in the set of datapoint_ids_ because of the:
```
  on_datapoint_update:
    - sensor_datapoint: AIRHSTAT
```
I have a TODO here. Ideally anything that is needed for `on_datapoint_update` with `datapoint_type: raw` will be in a different set so that we don't have to remove them here and we might have to request them on their own.

AIRHSTAT works for you because it happens to listen to it over the bus. We don't explicitly request it.

We don't have any sensors that get data from HWSTATUS. There was some old code but it was just logging so I removed it.